integration_test/cloudbuild-integration-test.yaml

```diff
@@ -291,6 +336,7 @@
   - "--quiet"
```

Comment:
Do you need to delete the second dcm store as well?

Reply:
Well, it won't break the test, since deleting pre-existing stores is part of setup.
But sure, makes sense to delete it here.